exercise/page.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Page</title>
+    <link rel="stylesheet" href="page.css">
+</head>
+<body>
+    <main class="card">
```

Comment:
I think it will be bette to replace main to other name
like
```suggestion
    <div class="card">
```
or
```suggestion
    <div class="containter">
```